tap_aggregator/src/server.rs

```diff
@@ -456,33 +464,36 @@ mod tests {
             .build(format!("http://127.0.0.1:{}", local_addr.port()))
             .unwrap();
 
-        // Create 100 receipts
+        // Create 300 receipts
         let mut receipts = Vec::new();
-        for _ in 1..100 {
+        for _ in 1..300 {
```

Comment:
Could we derive this 300 or at least change it to a variable? Right now it is not super clear why 100 or why it was changed to 300.

It might be more clear to do something like:

```rust
// Find the number of receipts needed to exceed the request size limit
let number_of_receipts_to_exceed_limit = http_request_size_limit / std::mem::size_of(Receipt) + 1;

for _ in 1..number_of_receipts_to_exceed_limit {
```

Reply:
`http_request_size_limit / std::mem::size_of(Receipt) + 1;` is not ideal, because I want to set a fixed number here that supports the maximum number of receipts defined in the spec/docs.

Reply:
`http_request_size_limit / std::mem::size_of(Receipt) + 1` is not ideal, because I want to set a fixed number here that supports the maximum number of receipts defined in the spec/docs.